src/lib/holocene/modal.svelte

```diff
@@ -44,10 +54,12 @@
 
   const handleCancel = () => {
     modalElement.close();
```

Comment:
Could we just call `close()` here instead?
```suggestion
    close();
```